### DISCUSSION TITLE
UltraRares util weird issue

### DISCUSSION THREAD

**Discussion by schoeffeljp** [nftchef/art-engine#202] [Help + Q&A]
Hi, has any one got that issue: "Error: Number of images and number of metadata JSON files do not match. "
That said, there are exactly the same number of images and metadata in the images and json folder. I used the -d and it indeed returns one extra image count than what's in there.
Running on Mac. Maybe it generates a system file, not sure.

**Comment by mikeheiser9** [comment]
I have this same issue

**Reply by mikeheiser9** [reply]
> I have this same issue

@imssarl did you ever solve this issue?

@nftchef if you have any insights into what could be happening here I would greatly appreciate some feedback. 

Everything works locally when I clone the repo. But as soon as I replace an image it gives me the error that there are not the same number of issues in both repos. There does not seem to be any untracked files in the repo, I do not have the same issue when I replace one of the JSON files in the ultraRare folder. So the issue seems to be with the images but I can't seem to figure out what the issue could be and I've tried everything I could think of at this point.


![Screen Shot 2022-12-17 at 12 50 53 AM](https://user-images.githubusercontent.com/35270182/208227571-8f5f1c1b-9ce6-4114-907b-985013c18a8c.png)


![Screen Shot 2022-12-17 at 12 17 20 AM](https://user-images.githubusercontent.com/35270182/208227572-ede721f2-d8d9-4550-8b79-9323f569bb1e.png)

Any insight from anyone on this issue would be very helpful.

**Reply by nftchef** [reply]
hi @mikeheiser9 this is generally caused by a hidden file in the ultraRares directory (parent) or the images folder, looks like the child directories are good though. 

If you still have the issue, can you please reach out to me on discord to try and debug the issue?
genkihagata#3074

**Reply by mikeheiser9** [reply]
> genkihagata#3074
@nftchef 
I am still having the issue I reached out on discord from megga#6364

**Reply by mikeheiser9** [reply]
For anyone having this issue run ls -a in the parent and subdirectories of your ultraRares repo. I had a DS_STORE in my images and parent repos. thanks @nftchef for your help!
